### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by aryakrekhi**

### CODE REVIEW

File: tfac/PCA_functions.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+
+def R2X_plot(datafile):
+  R2X_list = []
+  data = StandardScaler().Tensor_LINCS_MEMA(data)
+  for x in range(2, 21):
+      PCA_model = PCA(n_components=x)
+      PCA_model.fit(data)
+      R2X_list.append(np.sum(PCA_model.explained_variance_ratio_) * 100)
+  plt.plot(range(2, 21), R2X_list)
```
Comment (Farnazmdi) [inline tfac/PCA_functions.py:16]: you don't need to write an R2X function. You can use the one from tensorpack (take a look at figure1 to see how to use it.) I believe we don't need an R2X plot. Just run it for yourself to decide how many components you want. Although, you can decide from the reduction plot, too (figure1e).

File: tfac/PCA_functions.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+def scores_plot(datafile):
+    data = StandardScaler().Tensor_LINCS_MEMA(datafile)
+    PCA_model = PCA(n_components=2)
+    pca_scores = PCA_model.fit_transform(data)
+    plt.scatter(pca_scores[:,0], pca_scores[:,1])
```
Comment (Farnazmdi) [inline tfac/PCA_functions.py:22]: you can write the scores and loadings plot in the `plotHelpers.py` file.

File: tfac/PCA_functions.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+"""Contains function for performing PCA on OHSU data"""
+from dataHelpers import Tensor_LINCS_MEMA
+from sklearn.preprocessing import StandardScaler
```
Comment (Farnazmdi) [inline tfac/PCA_functions.py:3]: the data has already been normalized in the importing function, so you don't need to do it again.

**Review by Farnazmdi — CHANGES_REQUESTED** [review]
I strongly recommend running figure1 first (make output/figure1.svg) and taking a look at the plots to see where we are at.
Then you can use figure 1e to see how many components to use for PCA.
As I mentioned below, please write your scores and loadings plot function in the plotHelpers.py.
After these, we will focus on what to annotate and what to plot to see the repeating patterns.

**Comment by Farnazmdi** [comment]
@aryakrekhi please do `git pull` before making changes.

**Comment by Farnazmdi** [comment]
Please do `git pull`.
